### PR TITLE
CEPH-83575562: Tesco: RGW Segfaults on s3:copyObj with admin user

### PR DESCRIPTION
**Pull request by viduship**
…obj post

[Bug 2177220] [CEE/sd][RGW-Multisite][Ceph Upgrade] RGW crashes with Segmentation fault on s3:copy_obj post RHCS 5.3.1 upgrade
Polarian test case https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83575562
LOGS : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-M15RZ5/

### CODE REVIEW

File: rgw/v2/tests/aws/test_sts_rename_large_object.py [hunk 1]
```diff
@@ -111,6 +111,12 @@ def test_exec(config, ssh_con):
         )
     )
     utils.exec_shell_cmd(add_caps_cmd)
+    if admin_user:
```
Comment (ckulal) [inline rgw/v2/tests/aws/test_sts_rename_large_object.py:114]: From where do we get value for admin_user?

File: rgw/v2/tests/aws/test_sts_rename_large_object.py [hunk 2]
```diff
@@ -111,6 +111,12 @@ def test_exec(config, ssh_con):
...
+        add_admin_flag = (
+            'sudo radosgw-admin user modify add --uid="{user_id}" --admin true'.format(
```
Comment (ckulal) [inline rgw/v2/tests/aws/test_sts_rename_large_object.py:116]: Please check the command,
looks like "add" is invalid option..should be just modify?

Comment (viduship) [reply rgw/v2/tests/aws/test_sts_rename_large_object.py]: oh, that was a previous addition. Removed it @ckulal. request a re-review.

File: rgw/v2/tests/aws/test_sts_rename_large_object.py (hunk 1 above)
Comment (viduship) [reply rgw/v2/tests/aws/test_sts_rename_large_object.py]: Removed.

**Review by ckulal — COMMENTED** [review]
Looks good

**Review by anrao19 — APPROVED** [review]

**Review by TejasC88 — APPROVED** [review]

**Pull request merged**